airqo-platform/AirQo-frontend
# Ft sites registry

#### Summary of Changes (What does this PR do?)

- Applies sites summary api to sites table
- Refactor site details page to only retrieve a given site's details and not check the entire list of sites

- Country, Region and District to be added to the summary api

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?

- On deploy preview, visit /sites and the site details page. Both should load faster than production pages

#### What are the relevant tickets?

- [AN-409](https://airqoteam.atlassian.net/browse/AN-409) 

#### Screenshots (optional)
![image](https://github.com/airqo-platform/AirQo-frontend/assets/46527380/2361e85e-a362-497d-b01d-bb6a6fabeeb7)


[AN-409]: https://airqoteam.atlassian.net/browse/AN-409?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
